content/spin/cli-reference.md

```diff
@@ -6,145 +6,80 @@ enable_shortcodes = true
 url = "https://github.com/fermyon/developer/blob/main/content/spin/cli-reference.md"
 
 ---
-- [Spin](#spin)
-  - [Add](#add)
-  - [Build](#build)
-  - [Cloud](#cloud)
-    - [Deploy (Cloud)](#deploy-cloud)
-    - [Login (Cloud)](#login-cloud)
-  - [Deploy](#deploy)
-  - [Help](#help)
-  - [Login](#login)
-  - [New](#new)
-  - [Plugins](#plugins)
-    - [Install (Plugins)](#install-plugins)
-    - [List (Plugins)](#list-plugins)
-    - [Uninstall (Plugins)](#uninstall-plugins)
-    - [Update (Plugins)](#update-plugins)
-    - [Upgrade (Plugins)](#upgrade-plugins)
-  - [OCI Registry](#oci-registry)
-    - [Login (OCI Registry)](#login-oci-registry)
-    - [Pull (OCI Registry)](#pull-oci-registry)
-    - [Push (OCI Registry)](#push-oci-registry)
-  - [Templates](#templates)
-    - [Install (Templates)](#install-templates)
-    - [List (Templates)](#list-templates)
-    - [Uninstall (Templates)](#uninstall-templates)
-    - [Upgrade (Templates)](#upgrade-templates)
-  - [Up](#up)
-    - [Trigger Options](#trigger-options)
-      - [Redis Request Handler](#redis-request-handler)
-      - [HTTP Request Handler](#http-request-handler)
-  - [Watch](#watch)
-  - [CLI Stability Table](#cli-stability-table)
-
-## Spin
-
-This page documents the Spin Command Line Interface (CLI). Specifically, all of the available Spin Options and Subcommands. For more information on command stability, see the [CLI stability table](#cli-stability-table). You can reproduce the Spin CLI documentation on your machine by using the `--help` flag. For example:
+- [spin add](#spin-add)
+- [spin build](#spin-build)
+- [spin cloud](#spin-cloud)
+- [spin deploy](#spin-deploy)
+- [spin cloud deploy](#spin-cloud-deploy)
+- [spin login](#spin-login)
```

Comment:
I just noticed `deploy` and `login` are out of alphabetical order.  For findability, they should remain in order.